README.md

```diff
@@ -129,6 +129,12 @@ This feature can be controlled via the following variables:
 * `OPENJ9_SCC` (environment variable)
   *  Decription: If `"true"`, cache application-specific in an SCC and include it in the image. A new SCC will be created if needed, otherwise data will be added to the existing SCC.
   *  Default: `"true"`.
+* `TRIM_SCC` (environment variable)
```

Comment:
@mpirvu Apologies for the delay in getting to the review. Realized that the allowed values for `TRIM_SCC` (_yes_ or _no_) don't align with the values of `OPENJ9_SCC` (_true_ or _false_) or other build arguments. It'll be good to keep them consistent. Within the populate_scc.sh script its value can remain as yes/no, but the new build argument exposed to the user can be true/false.